turbinia/lib/docker_manager.py

```diff
@@ -150,20 +150,27 @@ def _create_mount_points(self, mount_paths, mode='rw'):
         list: The device blocks that will be mounted.
         dict: The file paths that will be mounted.
     """
+    accepted_vars = ['rw', 'ro']
     device_paths = []
     file_paths = {}
 
-    for mpath in mount_paths:
-      if mpath not in file_paths.keys() and mpath not in device_paths:
-        if IsBlockDevice(mpath):
-          formatted_path = '{0:s}:{0:s}:{1:s}'.format(mpath, 'r')
-          device_paths.append(formatted_path)
-        else:
-          file_paths[mpath] = {'bind': mpath, 'mode': mode}
+    if mode in accepted_vars:
+      for mpath in mount_paths:
+        if mpath not in file_paths.keys() and mpath not in device_paths:
```

Comment:
Actually, looking at this again, I don't think this would ever show up in the `device_paths` as is because what gets added to that variable is the formatted path and not the mpath.  You could create the formatted path above this and check for that instead.

Reply:
Great catch, updated it so that I format the path before the if statement then check against `device_paths` then.